Nos/Views/Home/HomeFeedView.swift

```diff
@@ -74,6 +74,12 @@ struct HomeFeedView: View {
                     .readabilityPadding()
                     .tipBackground(LinearGradient.horizontalAccentReversed)
                     .tipViewStyle(.inline)
+                    .onAppear {
+                        analytics.displayedWelcomeToFeedTip()
+                    }
+                    .onDisappear {
+                        analytics.dismissedWelcomeToFeedTip()
+                    }
```

Comment:
@mplorentz bringing the discussion about analytics into this thread. Above, you can see what I did, but as I'm sure you'll see, this means we may send the "welcome feed displayed" event many many many times, as well as the dismissed one. Maybe that's worth something, and maybe it's better than nothing.

Reply:
ok, let's not do this. I was thinking it would be nice to know what users do after seeing the tips. Do they follow them, do they stay in the app longer, etc. But we can't get that sort of data from this.